legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -167,19 +167,24 @@ def search_businesses():
         draft_query = db.session.query(Filing).filter(
```

Comment:
Would be nice to put these in a service, but just trying to do smallest changes possible here